tutorials/preparing-the-vm.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: Preparing the Virtual Machine
+---
+## Purpose
+This lesson describes the steps for configuring a virtual machine to run Kite example code on a Cloudera Quickstart VM.
+
+### Result
+Your VM is ready for you to install and run sample programs from the Kite SDK Examples project.
```

Comment:
I'd like this to be more specific. This is what happens, but the reason why we want these steps completed is to move on to the next tutorial. There should be a link here as well as a link towards the bottom.